src/WidgetBase.ts

```diff
@@ -168,7 +168,7 @@ export class WidgetBase<P extends WidgetProperties = WidgetProperties, C extends
 	/**
 	 * cachedVNode from previous render
 	 */
-	private _cachedVNode?: VNode | string;
+	private _cachedVNode?: (VNode | string | null)[] | VNode | string;
```

Comment:
We use `VNode | string | null` often enough to export this as a known type from `interfaces`, any one have a suggestion for a name?